bin/p2-exec/main.go

```diff
@@ -29,6 +34,8 @@ var (
 	nolim          = kingpin.Flag("nolimit", "Remove rlimits.").Short('n').Bool()
 	clearEnv       = kingpin.Flag("clearenv", "Clear all environment variables before loading envDir(s).").Bool()
 	workDir        = kingpin.Flag("workdir", "Set working directory.").Short('w').String()
+	umask          = kingpin.Flag("umask", "umask for this process and its subprocesses. Leave this off to use the system default. Use octal notation ex. 0022").Short('m').Default(umaskSentinal).String()
+	umaskSentinal  = "CannotParseAsNumeric"
```

Comment:
nit: I would like to suggest that a _sentinel_ value of `""` would fill the same roll and be a little easier to work with.